manifests/modelserving/caikit-ootb.yaml

```diff
@@ -42,8 +42,9 @@ objects:
         env:
           - name: RUNTIME_LOCAL_MODELS_DIR
             value: /mnt/models
+          - name: RUNTIME_GRPC_SERVER_THREAD_POOL_SIZE
+            value: "64"
```

Comment:
When deploying, it seems that we cannot have ints as env values, is this alright? @Xaenalt @andrewballantyne

Reply:
I never tested it on cluster, it was all a slack conversation... if you can't use ints, not sure you can do anything else -- so as long as it works out that's all that matters. @Xaenalt needs to make that call.

Reply:
Yeah, that's fine, shell interprets them the same

Reply:
But is that an statement or it's just a guess?

Reply:
Statement

Reply:
Great thanks!